app/controllers/usersController.js

```diff
@@ -64,21 +65,25 @@ exports.create = (req, res, next) => {
     });
 };
 
-// exports.listUsers = (req,res,next) => {
-//     let limit = 50;   // number of records per page
-//     let offset = 0;
-//     User.findAndCountAll()
-//     .then((data) => {
-//       User.findAll({
-//         attributes: ['name', 'surname', 'email'],
-//         limit: limit,
-//         offset: offset
-//       })
-//       .then((users) => {
-//         res.status(200).json({'result': users, 'count': data.count});
-//       });
-//     })
-//     .catch(function (error) {
-//       res.status(500).send('Internal Server Error');
-//     });
-//   };
+exports.getAllUsers = (req, res, next) => {
+  const limit = 10;
+  const offset = 0;
+  User.findAll({
+    attributes: ['name', 'surname', 'email'],
+    offset,
+    limit
+  })
+    .then(users => {
+      res.status(201);
+      res.send({ users });
+    })
+    .catch(err => {
+      if (err.errors) {
+        logger.error(err.message);
```

Comment:
you can delegate this stuff to the error middleware using the `next` function call.